src/TC/src/TaskDecodeFrame.cpp

```diff
@@ -605,7 +605,7 @@ struct FfmpegDecodeFrame_Impl {
          * To deal with that, discard PTS and make libavcodec come up with
          * correct auto-generated value.
          */
-        pkt->pts = AV_NOPTS_VALUE;
+        //pkt->pts = AV_NOPTS_VALUE; // What is that line used for? It prevents proper seeking
```

Comment:
There's a clear comment above this line that explains it's presence.

Reply:
Yes, I saw the clear comment above this line , but does not seem necessary in my tests, and prevents seeking (as stated in the clear comment on the side of this line :-)
(If that line is required, then we should find a way to restore correct pts)